### DISCUSSION TITLE
UIViewの配列表現について

### DISCUSSION THREAD

**Discussion by ShunHigu** [lit-iphone-course/lit-iphone-course#50] [Question-original]
１　写真を動かすときに、今後に写真同士の座標が同じになった時の動作などを加えたいので、配列で表したいです。
例えば、RendaImage[1]において
x座標はRendaImage[1][0]=120
y座標はRendaImage[1][1]=200
のように表したいです。

２　初めに「var RendaImage:[UIImageView]=()」のようにしておき、ifの中でRendaImage[1]ではStoneの写真、RendaImage[2]ではEnemyの写真が中に入っているようにしようと思いました。

３　画像を動かすコードはこれを参考にしました。
https://i-app-tec.com/ios/uiimageview.html

４　var RendaImage:[UIImageView]=()

５　https://github.com/ShunHigu/Original

**Comment by Yonpachi48** [comment]
いまいちどこで詰まっているのかが把握できてないんだけど、画像を動かして座標を取得するのは[これ](https://i-app-tec.com/ios/image-drag.html)を参考にしてみてください！
これで取得したX,Y座標の値と、素材構成ボックスのX,Y座標の値を比べて条件分岐を行えばできると思います！
